behind-the-veil-siteroot/imports/ui/components/pages/artist/AddEditPostPage.jsx

```diff
@@ -190,8 +198,12 @@ export const AddEditPostPage = ({ isEdit }) => {
           }
         );
       }
-    })
-      .catch((reason) => alert(reason));
+    }).then(() => {
```

Comment:
Just for future reference, I think you can avoid duplicating logic (i.e. `setIsSubmitting(false)`) in .then() and .catch() by including the logic in .finally()
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/finally

but if it's working rn then just leave it

Reply:
okiii noted!